package.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "kuzzle",
   "author": "The Kuzzle Team <support@kuzzle.io>",
-  "version": "2.10.0",
+  "version": "2.10.1",
   "description": "Kuzzle is an open-source solution that handles all the data management through a secured API, with a large choice of protocols.",
   "bin": {
-    "kuzzle": "./bin/start-kuzzle-server"
+    "kuzzle": "bin/start-kuzzle-server"
```

Comment:
:eyes:

Reply:
wat

Reply:
I just ran some tests: it looks like `npm version patch` also sanitizes the package.json file :thinking: 

I don't think that this changes anything. Do you want me to revert that change (made by NPM)?

Reply:
I upgraded to NPM7 a couple days ago, so it might be a new feature.

Reply:
Oh this was a NPM7 doing? :eyes: 
Go for it then, it would be modified by sanitization anyway.
:+1: